src/config/media/pipewire.md

```diff
@@ -0,0 +1,52 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+PipeWire should be started as a user. Use the autostarting mechanism of your
+desktop environment. If using [startx](../graphical-session/xorg.md#startx),
+add the following line in your `.xinitrc` file:
+
+```
+pipewire &
+```
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. To enable it, uncomment the following
+line in `/etc/pipewire/pipewire.conf` and restart the `pipewire` daemon:
+
+```
+exec /usr/bin/pipewire-pulse
+```
+
+If PulseAudio service is enabled,
+[disable](../services/index.md#disabling-services) it and reboot.
+
+To check if the replacement is working correctly, use the following command:
+
+```
+$ pactl info
+
+...
+Server Name: PulseAudio (on PipeWire 0.3.18)
+...
+```
+
+## ALSA integration
+
+Install `alsa-pipewire`, then enable the PipeWire ALSA device and make it
+default:
+
+```
+# mkdir -p /etc/alsa/conf.d
+# ln -s /usr/share/alsa/alsa.conf.d/50-pipewire.conf /etc/alsa/conf.d
+# ln -s /usr/share/alsa/alsa.conf.d/99-pipewire-default.conf /etc/alsa/conf.d
+```
+
+## JACK replacement
+
+Install `libjack-pipewire` and use `pw-jack` to launch JACK clients:
+
+```
+$ pw-jack <application>
+```
```

Comment:
here you could mention how to set up the automatic libjack override via `/etc/ld.so.conf.d` (glibc) or `/etc/ld-musl-$ARCH.path` (musl)

Reply:
I'm not sure what exactly this suggestion should be -- never worked with JACK.
I can see some notes on this in your gist, but I'm not sure what the actual steps should be. Can you give an example?

Reply:
@kkga I these steps are available [on Reddit](https://www.reddit.com/r/voidlinux/comments/mjzlry/replacing_jack_with_pipewire/gteamw8?utm_source=share&utm_medium=web2x&context=3).

For glibc:
1. Create `/etc/ld.so.conf.d/pipewire-jack.conf` with content `/usr/lib/pipewire-0.3/jack`
2. `sudo ldconfig`

Worked for me.

Reply:
Added a note for glibc.
Can you post the exact steps needed for musl? Don't want to mess this up, since I have no way of testing.

Reply:
I have just tried it for musl and it's too easy to fuck up, so I don't feel like recommending it. Basically musl will not fall back to `/lib` unless explicitly told so, so creating that file is dangerous.

Reply:
What is the problem exactly for musl? I remember it working effortlessly, although right now I am on glibc and cannot test.
I would prefer for the instructions to be complete with respect to musl as well.

Reply:
Writing a path to musl's config file and forgetting to also include `/lib` means your system won't find any libraries. And now you can't even turn off your computer because `sudo` doesn't work.